lib/Runtime/Language/InterpreterStackFrame.h

```diff
@@ -163,10 +163,16 @@ namespace Js
         int64* m_localInt64Slots;
         double* m_localDoubleSlots;
         float* m_localFloatSlots;
+        union
+        {
+            JavascriptArrayBuffer* m_asmJsBuffer;
+#ifdef ENABLE_WASM
+            WebAssemblyMemory * m_wasmMemory;
+#endif
+        };
```

Comment:
Where is this initialized?

Reply:
This is initialized in `InterpreterStackFrame::AlignMemoryForAsmJs` which is a necessary step to do asm.js and wasm in the interpreter

Reply:
I would still say you initialize this to nullptr in InitializeAllocation